src/queries/duckdb/seattle_roads_for_map.sql

```diff
@@ -5,7 +5,7 @@ COPY (
         level,
```

Comment:
`level` isn't an available column on transportation segments, so this query won't run as is.